red/runtime/locales/en-US/runtime.json

```diff
@@ -142,6 +142,7 @@
             "restore": "Restoring __type__ file backup : __path__",
             "restore-fail": "Restoring __type__ file backup failed : __message__",
             "fsync-fail": "Flushing file __path__ to disk failed : __message__",
+            "fwrite-fail": "Writing backup file __path__ to disk failed.",
```

Comment:
This should include the error message as the messages above do.

Reply:
Done